packages/react/src/utils/useCustomPages.tsx

```diff
@@ -25,6 +26,7 @@ export const useUserProfileCustomPages = (children: React.ReactNode | React.Reac
     reorderItemsLabels,
     LinkComponent: UserProfileLink,
     PageComponent: UserProfilePage,
+    MenuItemsComponent: MenuItems,
```

Comment:
Not sure why custom pages need to be aware of menu items. Can you elaborate ?

Reply:
Resolved in https://github.com/clerk/javascript/pull/3784#discussion_r1695462917